packages/xo-server/src/api/xosan.js

```diff
@@ -346,14 +346,12 @@ async function remoteSsh (glusterEndpoint, cmd, ignoreError = false) {
         }
       }
     }
-    debug(
-      result.command.join(' '),
-      '\n  =>exit:',
-      result.exit,
-      '\n  =>err :',
-      result.stderr,
-      '\n  =>out (1000 chars) :',
-      result.stdout.substring(0, 1000)
+
+    log.debug(
+      `${result.command.join(' ')} =>exit: ${result.exit} =>err: ${
+        result.stderr
+      } =>out (1000 chars) : ${result.stdout.substring(0, 1000)}`,
+      { result }
```

Comment:
I'm afraid `result` could be quite big, IMHO it would be best to do:

```js
log.debug(`result of ${result.command.join(' ')}`, {
  exit: result.exit,
  err: result.stderr,
  out: result.stdout.substring(0, 1000),
})
```